### PR TITLE
WINUI-3339-Getting-started-UG-AvatarView

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Addition of Getting Started Page for Avatar view UserGuide(UG).

### CODE REVIEW

File: winui/AvatarView/getting-started.md [hunk 1]
```diff
@@ -1,10 +1,80 @@
 ---
 layout: post
-title: Getting Started with WinUIAvatar View control | Syncfusion
+title: Getting Started with WinUI AvatarView control | Syncfusion
```
Comment (Manivannan-E) [inline winui/AvatarView/getting-started.md:3]: @MadhumithanJayapathi Learn here all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.

File: winui/AvatarView/getting-started.md [hunk 2]
```diff
@@ -1,10 +1,80 @@
...
 control: AvatarView
 documentation: ug
 ---
 
-# Getting Started with WinUI AvatarView
\ No newline at end of file
+# Getting Started with WinUI AvatarView
+
+This section explains the steps required to add the WinUI AvatarView control and covers only the basic features needed to get started with Syncfusion `AvatarView` control.
+
```
Comment (Manivannan-E) [inline winui/AvatarView/getting-started.md:13]: @MadhumithanJayapathi This section explains the steps required to add the WinUI AvatarView control with its basic features.

File: winui/AvatarView/getting-started.md [hunk 3]
```diff
@@ -1,10 +1,80 @@
...
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+
+//Setting page content.
+page.Content = avatarView;
+           
+{% endhighlight %}
+{% endtabs %}
+
+![AvatarView control](avatarview_images/winui_avatarview_initialise.png)
+
```
Comment (Manivannan-E) [inline winui/AvatarView/getting-started.md:55]: @MadhumithanJayapathi change WinUI AvatarView control and change the image name to winui_avatarview

Comment (MadhumithanJayapathi) [reply winui/AvatarView/getting-started.md:55]: Changeg to WinUI AvatarView control and changed the image name to winui_avatarview

File: winui/AvatarView/getting-started.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/getting-started.md:13]: @Manivannan-E Changed the explanation as mentioned.

File: winui/AvatarView/getting-started.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/AvatarView/getting-started.md:3]: @Manivannan-E  Changed the summary as mentioned

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/AvatarView/getting-started.md [hunk 4]
```diff
@@ -1,10 +1,77 @@
 ---
 layout: post
-title: Getting Started with WinUIAvatar View control | Syncfusion
-description: Learn all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more here.
+title: Getting Started with WinUI AvatarView control | Syncfusion
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:3]: Title: Getting Started with WinUI AvatarView control | Syncfusion

File: winui/AvatarView/getting-started.md [hunk 5]
```diff
@@ -1,10 +1,77 @@
...
+description: Learn here all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:4]: Description: Learn about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control with its basic features.

File: winui/AvatarView/getting-started.md [hunk 6]
```diff
@@ -1,10 +1,77 @@
...
+## Creating an application with WinUI AvatarView
+
+1. Create a [WinUI 3 desktop app for C# and .NET 6](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/get-started-winui3-for-desktop).
+2. Add reference to [Syncfusion.Core.WinUI](https://www.nuget.org/packages/Syncfusion.Core.WinUI) NuGet. 
+3. Import the control namespace `Syncfusion.UI.Xaml.Core` in XAML or C# code.
+4. Initialize the `SfAvatarView` control.
+
+## Initialize AvatarView 
+
+By default, AvatarView control is displayed with `Avatar1` character image, AvatarShape of `Circle and AvatarSize of `Large`.
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:23]: By default, the AvatarView control is displayed with the `Avatar1` character image, the AvatarShape of `Circle,` and the AvatarSize of `Large.`

File: winui/AvatarView/getting-started.md [hunk 7]
```diff
@@ -1,10 +1,77 @@
...
+SfAvatarView avatarView = new SfAvatarView();
+           
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control](avatarview_images/winui_avatarview.png)
+
+## Initialize AvatarView with ImageSource
+
+You can add any custom image as an avatar in WinUI AvatarView control using the `ImageSource` property.
```
Comment (Chrispineagunja) [inline winui/AvatarView/getting-started.md:56]: Add any custom image as an avatar in WinUI AvatarView control using the `ImageSource` property.

**Pull request merged**